pytensor/link/numba/dispatch/slinalg.py

```diff
@@ -266,27 +158,32 @@ def impl(A, B, trans=0, lower=False, unit_diagonal=False):
             INFO,
         )
 
+        _solve_check(int_ptr_to_val(LDA), int_ptr_to_val(INFO))
+
         if B_is_1d:
-            return B_copy[..., 0], int_ptr_to_val(INFO)
-        return B_copy, int_ptr_to_val(INFO)
+            return B_copy[..., 0]
+
+        return B_copy
 
     return impl
 
 
 @numba_funcify.register(SolveTriangular)
 def numba_funcify_SolveTriangular(op, node, **kwargs):
-    trans = op.trans
+    trans = bool(op.trans)
     lower = op.lower
     unit_diagonal = op.unit_diagonal
     check_finite = op.check_finite
+    overwrite_b = op.overwrite_b
+    b_ndim = op.b_ndim
 
     dtype = node.inputs[0].dtype
     if str(dtype).startswith("complex"):
         raise NotImplementedError(
             "Complex inputs not currently supported by solve_triangular in Numba mode"
         )
 
-    @numba_basic.numba_njit(inline="always")
+    @numba_basic.numba_njit(inline="always", cache=True)
```

Comment:
We have a default handling of cache in the `numba_njit` already, why this explicit?

Reply:
I was playing around with settings and forgot to delete this. Good catch.